app/controllers/developers_controller.rb

```diff
@@ -45,25 +45,14 @@ def update
   end
 
   def show
-    if (developer = developer_to_redirect_to_hashid)
-      return redirect_to developer, status: :moved_permanently,
-        notice: t(".redirection", url: developer_url(developer))
```

Comment:
The test is failing because there is now an unused translation. Removing this from `config/locales/en.yml` should fix it!

```
+--------+-----------------------------+----------------------------------------------------------------------------------+
[13](https://github.com/joemasilotti/railsdevs.com/actions/runs/4093358785/jobs/7058739083#step:11:14)
| Locale | Key                         | Value                                                                            |
[14](https://github.com/joemasilotti/railsdevs.com/actions/runs/4093358785/jobs/7058739083#step:11:15)
+--------+-----------------------------+----------------------------------------------------------------------------------+
[15](https://github.com/joemasilotti/railsdevs.com/actions/runs/4093358785/jobs/7058739083#step:11:16)
|   en   | developers.show.redirection | Links to developer profiles have changed. Please update your bookmark to %{url}. |
[16](https://github.com/joemasilotti/railsdevs.com/actions/runs/4093358785/jobs/7058739083#step:11:17)
+--------+-----------------------------+----------------------------------------------------------------------------------+
[17](https://github.com/joemasilotti/railsdevs.com/actions/runs/4093358785/jobs/7058739083#step:11:18)
```

You can check this locally by running `bundle exec i18n-tasks health en`.

Reply:
Thanks! Updated!